portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
+body {
+    background-color: lightblue;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    height: 100%;
+    font-family: 'Chelsea Market', cursive;
+}
```

Comment:
good practice is to make the properties in alphabetical order.
same fore below

Reply:
oh awesome! I will update and change that style habit

Reply:
seems like some are still not in alphabetical order. reopening.

Reply:
For things that are related such as height/width, flex stuff, etc. Can we group them next to each other or do you still keep it alphabetical?

Reply:
google's convention is to make it alphabetical